jena-core/src/test/java/org/apache/jena/reasoner/test/TestRDFSReasoners.java

```diff
@@ -95,6 +95,7 @@ public static TestSuite suite() {
     /**
      * Build a single named query test
      */
+    @SuppressWarnings("unused")
     private static void constructSingleQuerytests(TestSuite suite, String manifest, String test, ReasonerFactory rf, Resource config) throws IOException {
```

Comment:
PRs are "live" - if the your branch changes so does the PR.

If you edit your branch and "push" it will change here as well.

To get rid of it, edit the file, then "git squash" or "git add" then "git commit --amend" followed by "git push --force" and it will disappear from the PR.

Or I can clean it after pulling it. This isn't a huge piece of work.